pkg/caveats/types/basic.go

```diff
@@ -73,7 +73,7 @@ func convertNumericType[T int64 | uint64 | float64](value any) (any, error) {
 }
 
 var (
-	AnyType     = registerBasicType("any", cel.AnyType, func(value any) (any, error) { return value, nil })
```

Comment:
Is this a breaking change? is it possible this would render existing serialized CEL AST incompatible?

Reply:
No